### PR TITLE
Add auto-submit on copy from clipboard, refine code extraction

### PR DESCRIPTION
**Pull request by vrazn**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -85,17 +85,19 @@ export default class OTPInputView extends Component {
...
+        const regexp = new RegExp(`(\\d{${pinCount}})`, 'g')
+        Clipboard.getString().then(string => {
+            const otp = regexp.exec(string);
+            if (this.hasCheckedClipBoard && otp && (this.clipBoardCode !== otp[0])) {
                 this.setState({
-                    digits: code.split(""),
+                    digits: otp[0].split(""),
                 }, () => {
                     this.blurAllFields()
-                    this.notifyCodeChanged()
```
Comment (ansonyao) [inline index.js]: This line should stay I believe. Both callback should be fired.

File: index.js [hunk 2]
```diff
@@ -85,17 +85,19 @@ export default class OTPInputView extends Component {
...
+                    onCodeFilled(otp[0]);
```
Comment (ansonyao) [inline index.js]: onCodeFilled is optional. I suggest `onCodeFilled && onCodeFilled(result)`

**Comment by fpena** [comment]
Hi @vrazn . Would you please rebase/merge with `master`? 
Thanks!

**Comment by fpena** [comment]
Hi there @vrazn. Are you still interested in merging this PR? 
Please rebase with `master` branch. Thanks!

**Comment by BeckyWu220** [comment]
Hi @vrazn We appreciate your effort in submitting the PR. Since we haven't hear back from your for the rebasing, and PR https://github.com/Twotalltotems/react-native-otp-input/pull/58 that resolves the same issue had been merged. We are closing this PR. Thank you.

**Pull request closed without merging**